src/components/Button/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { cx } from "emotion"
+import {
+  wrapper,
+  primaryBtn,
+  secondaryBtn,
+  dangerBtn,
+  ghostBtn
+} from "./style.js"
+
+const ButtonPropTypes = {
+  className: PropTypes.string,
+  onClick: PropTypes.func,
+  disable: PropTypes.bool,
+  type: PropTypes.oneOf(["primary", "secondary", "danger"]),
+  fullWidth: PropTypes.bool,
+  size: PropTypes.oneOf(["small", "large"]),
+  ghost: PropTypes.bool,
+  href: PropTypes.string
+}
+
+function SubmitButton({ className, onClick = () => {}, size, href }) {
+  return (
+    <button
+      href={href}
+      onClick={(e) => {
+        e.preventDefault()
+        onClick(e.target.value)
+      }}
+      className={cx(
+        wrapper,
+        primaryBtn,
+        { small: size === "small" },
+        className
+      )}>
+      Submit
+    </button>
+  )
+}
+SubmitButton.propTypes = ButtonPropTypes
+
+function LinkButton({
+  className,
+  onClick = () => {},
+  type,
+  size,
+  ghost,
+  href
+}) {
+  return (
+    <a
+      href={href}
+      onClick={(e) => {
+        e.preventDefault()
+        onClick(e.target.value)
+      }}
+      className={cx(
+        wrapper,
+        { type: type === "primary" },
+        { small: size === "small" },
+        className
+      )}>
+      {}
+    </a>
+  )
+}
+LinkButton.propTypes = ButtonPropTypes
+export default SubmitButton
```

Comment:
I suggest having a single Button component instead of 2. We can distinguish the two Buttons using the type property. If `type="submit"` we render the SubmitButton. If type="link" we will render the LinkButton. 
The rest of the properties will be passed to each component. eg.
```jsx
const Button = ({ type, ...restProps }) => 
    type === "submit" ? <SubmitButton {...restProps} /> : <LinkButton {...restProps} />
```

We can rename the current type property to `variant` in order to display the "primary", "secondary", "danger" variations